From 21e28680d4a335802ed47d81dce8277081ed7935 Mon Sep 17 00:00:00 2001
From: runtime-promotion-bot-noreply
 <runtime-promotion-bot-noreply@jetbrains.com>
Date: Tue, 21 Apr 2026 04:15:46 +0000
Subject: [PATCH] IJI-3336 Updating versions of project dependencies:
 runtimeBuild->25.0.2b329.117

GitOrigin-RevId: 50775dbec8e845035f7ec401b5fc609a34047061
---
 build/dependencies/dependencies.properties | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/dependencies/dependencies.properties b/build/dependencies/dependencies.properties
index 3a32f41ae6d8..5d9f9935c27b 100644
--- a/build/dependencies/dependencies.properties
+++ b/build/dependencies/dependencies.properties
@@ -7,6 +7,6 @@ jdkBuild=21.0.10b1163.108
 launcherBuild=261.19843
 nsisBuild=3.11-1
 restarterBuild=243.6005
-runtimeBuild=25.0.2b329.111
+runtimeBuild=25.0.2b329.117
 snapDockerImage=registry.jetbrains.team/p/ij/docker-hub/jetbrains/snapcraft@sha256:0059496b7a941e27937bc523621a336bddc1327de8dcd03f6b0bd0b57892ce43
 toolboxBuild=3.2.0.65851
-- 
2.53.0

